innopoints/schemas/activity.py

```diff
@@ -1,9 +1,14 @@
+"""Schema for the Activity and Competence models."""
+
 from marshmallow import validate, validates_schema, ValidationError, pre_load, post_dump
 
 from innopoints.extensions import ma, db
 from innopoints.models import Activity, Application, ApplicationStatus, Competence
 from .application import ApplicationSchema
 
+
+# pylint: disable=missing-docstring
```

Comment:
Maybe you should just add it to pylintrc?
* would've been much better if https://github.com/PyCQA/pylint/issues/618 was resolved

Reply:
No, we want docstrings in many other places, just not schemas